packages/zowe-explorer-api/src/fs/types/datasets.ts

```diff
@@ -33,7 +34,7 @@ export const DS_EXTENSION_MAP: Map<string, (string | RegExp)[]> = new Map([
 
 export class DsEntry extends FileEntry implements DsEntryProps {
     public metadata: DsEntryMetadata;
-
+    public encoding?: ZosEncoding;
```

Comment:
This `encoding` property is already defined in the `FileEntry` class, I think we should avoid re-defining it here.